log-collector-script/linux/eks-log-collector.sh

```diff
@@ -346,6 +346,20 @@ get_iptables_info() {
     ip6tables-save > "${COLLECT_DIR}"/networking/ip6tables-save.txt
   fi
 
+  if ! command -v ipvsadm && command -v ipset > /dev/null 2>&1; then
+    echo "IPVS Linux kernel module not installed" | tee ipvsadm.txt ipset.txt
+  else
+    # check that ip_vs module is loaded in get_modinfo()
+    try "collect ipvs information"
+    ipvsadm --save | tee -a "${COLLECT_DIR}"/networking/ipvsadm.txt && sed -i '1s/^/add \tproto \tvirt-server \tscheduler algthm \treal-server \n/' "${COLLECT_DIR}"/networking/ipvsadm.txt
+    ipvsadm --list --numeric --rate | tee -a "${COLLECT_DIR}"/networking/ipvsadm.txt
+    ok -e "\n" | tee -a "${COLLECT_DIR}"/networking/ipvsadm.txt
+    ipvsadm --list --numeric --stats --exact | tee -a "${COLLECT_DIR}"/networking/ipvsadm.txt
+    ipset --list | tee -a "${COLLECT_DIR}"/networking/ipset.txt
```

Comment:
Same here, what if there have an old `"${COLLECT_DIR}"/networking/ipset.txt` existed for a long running node.

Reply:
Acknowledge. Updating tee to overwrite

Reply:
also there is a cleanup func in the script, that removes the entire {COLLECT_DIR} used for work, I believe

Reply:
@cartermckinnon @guessi do I need to replay this change on the latest base branch changes?